docs/src/updated_example.md

```diff
@@ -0,0 +1,111 @@
+# Uprading docs from Documenter.jl to DocumenterVitepress.jl
+
+Assuming that your current documentation is based on Documenter.jl, you can upgrade to DocumenterVitepress.jl by following these steps:
+
+1. Install DocumenterVitepress.jl using the following julia command in your package's environment:
+
+```julia
+using Pkg
+Pkg.add("DocumenterVitepress")
+```
+
+2. Now go the the `make.jl` file in your `docs` folder and do the following necessary changes to upgrade to DocumenterVitepress.jl :
+
+
+   a. The `make.jl` file with `Documenter,jl` look like this:
+
+```julia
+using OMOPCDMPathways
+using Documenter
+
+DocMeta.setdocmeta!(test, :DocTestSetup, :(using test); recursive=true)
+
+makedocs(;
+    modules=[test],
+    authors="jay-sanjay <landgejay124@gmail.com> and contributors",
+    sitename="OMOPCDMPathways.jl",
+    format=Documenter.HTML(;
```

Comment:
this will not work. You need to pass DocumenterVitepress to `format`.

Reply:
Ohh this is the part of the original make.jl from Documenter.jl and not DocumenterVitepress.jl

Reply:
DocumenterVitepress supports a `diff` language which might be useful in this case, it formats a git diff like you would see on Github.  I can always add that later though, just thought it would be useful to note.